lib/collection/set.js

```diff
@@ -26,9 +27,6 @@ export class SetColl extends Collection {
   }
 
   _addWithoutObserver(item) {
-    if (!item && item !== 0) {
-      throw "null objects are not allowed";
-    }
```

Comment:
Why did you remove this? This was deliberate.

I understand that JS Set technically allows this, but I don't want to allow in a Set. (It's OK in array and map, there are good usecases there.)

Reply:
You're also disallowing all other falsy values other than the `0` that you explicitly check for. If you really want to disallow `null`, you should check for `null`.

Reply:
(And a comment to indicate that this is different behaviour to array and map wouldn't go amiss, of course.)

Reply:
Agreed on both points. Only null and undefined should be forbidden. Disallowing `false` and `0` was accidental, indeed. I thought I had fixed that, but maybe I didn't. Would you like to fix it? And add the documentation as JSDoc?